apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -366,8 +367,14 @@ suite('Information view', function() {
           sizeContent
         );
       }
-
-      sinon.assert.called(reportView.renderContactList);
+      if (receivers) {
+        sinon.assert.calledWithMatch(
+          reportView.renderContactList,
+          receivers.concat('sender')
+        );
+      } else {
+        sinon.assert.calledWith(reportView.renderContactList);
+      }
```

Comment:
hey, here is what I had in mind: https://github.com/julienw/gaia/commit/593c6b2b81b7ccaf79723add7c0b582b366d950b

I noticed something not very good happening in these tests: renderContactList can actually be called twice, once in the setup (because we call beforeEnter) and one later on (because we call beforeEnter once again). I don't think this is really important but we'll need to fix this eventually.